.coveragerc

```diff
@@ -1,3 +1,8 @@
 [report]
-omit =
-    mesa/tests/*
+omit = 
```

Comment:
This cuts on the tests that happen, so there isn't extra cruft in our coveralls reports.